bpfd-operator/config/bpfd-deployment/config.yaml

```diff
@@ -7,7 +7,10 @@ data:
   ## Can be configured at runtime
   bpfd.agent.image: quay.io/bpfd/bpfd-agent:latest
   bpfd.image: quay.io/bpfd/bpfd:latest
-  bpfd.log.level: "info"
+  ## Can be set to "info", "debug", or "trace"
+  bpfd.agent.log.level: "info"
+  ## See https://docs.rs/env_logger/latest/env_logger/ for configuration options
+  bpfd.log.level: "bpfd=info"
```

Comment:
Is there a reason they aren't consistent?
```
bpfd.agent.log.level: "info"
bpfd.bpfd.log.level: "info"
```
OR
```
bpfd.log.level: "bpfd=info,agent=info"
```

Reply:
So with "bpfd=info" I was specifying to the rust process ONLY log for the bpfd crate at the info level.

I changed it for now to just be info....but to cut down on some of the dependency log verbosities when trying to debug it's often useful to do `bpfd=debug` for `bpfd.log.level`

However I want to leave the logging fully configurable to help users pinpoint problems (i.e `aya=debug` is also a valid configuration option)

Reply:
I'm going to go ahead and merge it, but the `aya=debug` should probably be in the docs too.